README.md

```diff
@@ -1,8 +1,8 @@
-# EU Login
+# OpenEuropa Auth
 
-[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/eu_login/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/eu_login)
+[![Build Status](https://drone.fpfis.eu/api/badges/openeuropa/oe_auth/status.svg?branch=master)](https://drone.fpfis.eu/openeuropa/oe_auth)
 
-The EU Login module allows to authenticate against the EU Login service.
+The OE Auth module allows to authenticate against the European Commission login service.
```

Comment:
Shall we maybe use the full name every time we refer to the module? Like `OpenEuropa Auth`